src/main/java/hudson/plugins/emailext/ExtendedEmailPublisherDescriptor.java

```diff
@@ -65,21 +65,10 @@ public final class ExtendedEmailPublisherDescriptor extends BuildStepDescriptor<
      */
     private String defaultSuffix;
 
-    /**
-     * Jenkins's own URL, to put into the e-mail.
-     */
-    private transient String hudsonUrl;
```

Comment:
since 87d418d0133c5dde080ae513e8dd50d4a66fa2b2

Reply:
Sadly....

[testChangeLogDisplayShouldntOnlyRelyOnPortableMethods2](https://github.com/jenkinsci/email-ext-plugin/blob/83f9b61637ce54c20603f1d67097b2217c4641a8/src/test/java/hudson/plugins/emailext/plugins/content/JellyScriptContentTest.java#L48)

Reply:
And why was ca20d1a52d159066920a7e414c1928d2cad24b2c accessing this field via reflection?

Reply:
Fixed in 2b18a5ce, whatever that was for.